src/composables/aeSdk.ts

```diff
@@ -45,24 +42,28 @@ import { useModals } from './modals';
 let aeSdk: AeSdkSuperhero;
 let aeSdkBlocked = false;
 let storedNetworkName: string;
-const isAeSdkReady = ref(false);
+
 const nodeNetworkId = ref<string>();
+
+const isAeSdkReady = ref(false);
+const isAeNodeReady = ref<boolean>(false);
+const isAeNodeConnecting = ref<boolean>(false);
+const isAeNodeError = ref<boolean>(false);
```

Comment:
<boolean> is not needed

Reply:
Why? It is used in the connection status bar.

Reply:
"<boolean>" is  not needed

I don't know this type is missing in comment

Reply:
"< boolean >" is  not needed

I don't know this type is missing in comment

Reply:
OK, makes sense. Please be more clear next time :)